Sources/Snowplow/Snowplow.swift

```diff
@@ -342,9 +342,11 @@ public class Snowplow: NSObject {
     /// - Parameter webViewConfiguration: Configuration of the Web view to subscribe to events from
     @objc
     public class func subscribeToWebViewEvents(with webViewConfiguration: WKWebViewConfiguration) {
-        let messageHandler = WebViewMessageHandler()
+        let messageHandlerOld = WebViewMessageHandler()
+        let messageHandlerV2 = WebViewMessageHandlerV2()
 
-        webViewConfiguration.userContentController.add(messageHandler, name: "snowplow")
+        webViewConfiguration.userContentController.add(messageHandlerOld, name: "snowplow")
+        webViewConfiguration.userContentController.add(messageHandlerV2, name: "snowplow")
```

Comment:
Shouldn't we use a different name for the message handler than `snowplow` since that is taken by the old handler? [This is](https://github.com/snowplow-incubator/snowplow-webview-tracker/blob/main/src/api.ts#L161) how it is called from the WebView tracker – if they have the same name, you won't be able to differentiate them.